terraform-modules/aws/istio-networking/main-gateway/main.tf

```diff
@@ -1,3 +1,12 @@
+terraform {
+  required_providers {
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = ">= 1.7.0"
+    }
+  }
+}
+
```

Comment:
Actually...we dont want to include this in the module b/c the user might want to include this in their usage on a global level.